docs/_docs/examples/func-replace.md

```diff
@@ -10,6 +10,8 @@ change field with a value derived from a function. This example
 was derived based on a prompt in [this pull request](https://github.com/pydicom/contrib-pydicom/pull/14).
 If you are interested in the code for this example, it's available
 [here](https://github.com/pydicom/deid/tree/master/examples/dicom/header-manipulation).
+If you are interested in the functions provided by deid (and you don't want to write your
+own function) see [this documentation](https://pydicom.github.io/deid/user-docs/recipe-groups/).
```

Comment:
The link on "this documentation" is currently referencing recipe-groups, this should be a link to recipe-funcs.

Reply:
That's called me failing at copy paste :laughing:

Reply:
Fixed.